arch/arm/soc/st_stm32/stm32f1/Kconfig.series

```diff
@@ -12,6 +12,7 @@ config SOC_SERIES_STM32F1X
 	select SOC_FAMILY_STM32
 	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
 	select HAS_STM32CUBE
+	select CPU_HAS_MPU
```

Comment:
Since MPU is not suported on all SoCs of stm32f1 series, this should be moved (if indeed supported) under Kconfig.defconfig.stm32f10...